Lib/fontbakery/profiles/adobefonts.py

```diff
@@ -106,4 +117,30 @@ def com_adobe_fonts_check_find_empty_letters(ttFont):
 # ToDo: add many more checks...
 
 profile.auto_register(globals())
+
+com_adobe_fonts_check_dsig = profile.check_log_override(
+    'com.google.fonts/check/dsig'
+  , 'com.adobe.fonts/check/dsig'
+  , reason='For Adobe this issue is not as severe '\
+            + 'as assessed in the original check.'
+  , overrides = (
+        (   FAIL # override_target -> Status or a specific Message.code (string)
+          , WARN # new_status -> None: keep old status
+          , None # new_message_string -> None: keep old message
+          )
```

Comment:
This needs to explicitely use a log-message keyword otherwise the over-ride is ambiguous and can result in problems in the future once the original check is updated with the addition of other log-messages.

See discussion in #2558 and, in particular, this specific comment: https://github.com/googlefonts/fontbakery/issues/2558#issuecomment-506483074

Reply:
It's bad style, but the check is not implemented that way currently. The code could use a message.code.

Reply:
LOL, yeah ... I should have rebased my working branch earlier.